advanced/Templates/gravity.db.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE whitelist (domain TEXT UNIQUE NOT NULL, enabled BOOLEAN DEFAULT 1, comment TEXT, dateadded DATETIME);
+CREATE TABLE blacklist (domain TEXT UNIQUE NOT NULL, enabled BOOLEAN DEFAULT 1, comment TEXT, dateadded DATETIME);
+CREATE TABLE regex     (filter TEXT UNIQUE NOT NULL, enabled BOOLEAN DEFAULT 1, comment TEXT, dateadded DATETIME);
+CREATE TABLE gravity   (domain TEXT UNIQUE NOT NULL);
+
```

Comment:
We should add a table to keep track of the database version. We may want to update the schema at some point, so we need to version it.